__tests__/endpoints.test.js

```diff
@@ -55,6 +55,22 @@ describe('Test Endpoints', () => {
                     expect(body.article_id).toBe(1);
                     expect(typeof body.title).toBe('string');
```

Comment:
I would test for all the properties that you expect when receiving an article object